post/views.py

```
@@ -0,0 +1,85 @@
+             title = request.data.get('title')
+             content=request.data.get('content')
+             if not title or not content:
+                  return Response({
+                  "detail": "[title,content] fields missing."
+                  },status=status.HTTP_400_BAD_REQUEST)
+             post.title=title
+             post.content=content
+             post.save()
+             serializer = PostSerializer(post)
+             return Response(serializer.data, status=status.HTTP_200_OK)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)